src/flag.ts

```diff
@@ -81,41 +82,255 @@ export const monikaFlagsDefaultValue: MonikaFlags = {
   flush: false,
   'follow-redirects': 21,
   force: false,
+  ignoreInvalidTLS: false,
   'keep-verbose-logs': false,
   logs: false,
   'one-probe': false,
+  output: 'monika.yml',
   repeat: 0,
   retryInitialDelayMs: 2000,
   retryMaxDelayMs: 30_000,
   // default is 20s interval lookup
   stun: 20,
   summary: false,
+  'symon-api-version': SYMON_API_VERSION.v1,
   symonGetProbesIntervalMs: Number.parseInt(
     process.env.FETCH_PROBES_INTERVAL ?? '60000',
     10
   ),
+  symonReportInterval: 10_000,
+  symonReportLimit: 100,
```

Comment:
Moving defaults here makes them magic number. Better name it like `DEFAULT_SYMON_REPORT_INTERVAL_MS` or something for clarity.

Reply:
Updated in [cc222d3](https://github.com/hyperjumptech/monika/pull/1277/commits/cc222d3ac54a6d019a8cbde1b412badaed9394bd). Thanks!